lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -24,7 +24,7 @@ class MarriageAbroadDataQuery
 
     OS_CONSULAR_CNI_IN_NEARBY_COUNTRY = %w(nicaragua)
 
-    OS_AFFIRMATION_COUNTRIES = %w(belgium cambodia colombia ecuador egypt lebanon finland macao mongolia morocco norway peru qatar south-korea united-arab-emirates vietnam)
```

Comment:
Not related to your change here, but its unclear to me what OS_AFFIRMATION_COUNTRIES means, can you clarify?